### ISSUE TITLE
Property and Query for Mediawiki "Summary" field when updating pages

### ISSUE THREAD

**Issue by rwjorgen** [SemanticMediaWiki/SemanticMediaWiki#4920]
Would it be possible to assign a Semantic Property to the "Change Summary" field that is completed when changing wiki pages and displayed on the Version History page?

I'd like to run a query that shows "changes since" a specific date, and then list the "Summary" field to provide a report that gives an overview of what was changed in the latest version.

**Comment by kghbln** [comment]
Currently this is not possible. Thanks for suggesting this enhancement.

**Comment by rwjorgen** [comment]
Yeah, I could not figure out how to do it, so I figured it would have to be a future enhancement (unless someone had a clever trick up their sleeve.)  I look forward to a future version that enables this capability!  Thanks!

**Comment by gesinn-it** [comment]
How about querying the API like `https://www.mediawiki.org/w/api.php?action=query&prop=revisions&titles=Main%20Page&rvlimit=5&rvprop=timestamp%7Cuser%7Ccomment` in combination with Extension:External Data?
